tests/unit/insertTestResult.unitTest.ts

```diff
@@ -1411,26 +1408,32 @@ describe("insertTestResult", () => {
 
             testResultsService = new TestResultsService(new MockTestResultsDAO());
 
-            expect.assertions(3);
-            return testResultsService.insertTestResult(testResultWithLecTestTypeWithoutCertNum)
-                .catch((error: { statusCode: any; body: any; }) => {
-                    expect(error).toBeInstanceOf(HTTPError);
-                    expect(error.statusCode).toEqual(400);
-                    expect(error.body).toEqual(ERRORS.NoCertificateNumberOnLec);
+            expect.assertions(7);
+            return testResultsService.insertTestResult(testResultLEC)
+                .then((insertedTestResult: any) => {
+                    expect(insertedTestResult[0].testTypes[0].testTypeId).toEqual("44");
+                    expect(insertedTestResult[0].testTypes[0].certificateNumber).toBe(null);
+                    expect(insertedTestResult[0].testTypes[0].testExpiryDate).toBeTruthy();
+                    expect(insertedTestResult[0].testTypes[0].modType).toBeTruthy();
+                    expect(insertedTestResult[0].testTypes[0].emissionStandard).toBeTruthy();
+                    expect(insertedTestResult[0].testTypes[0].smokeTestKLimitApplied).toBeTruthy();
+                    expect(insertedTestResult[0].testTypes[0].fuelType).toBeTruthy();
                 });
         });
-    });
 
-    context("when inserting a testResult that has an LEC testType with a certificateNumber", () => {
-        it("then the inserted test result should use the certificateNumber provided in post", () => {
```

Comment:
Is this no longer true? I notice the replacement test has certificateNumber === null

Reply:
It's no longer true and it's not specified on the BE ticket but the change is required on the FE ticket. The submission is breaking if I don't remove this certiificateNumber because the FE doesn't sent it anymore.